internal/rbd/clone.go

```diff
@@ -145,7 +145,7 @@ func (rv *rbdVolume) createCloneFromImage(ctx context.Context, parentVol *rbdVol
 	}
 
 	if parentVol.isEncrypted() {
-		err = parentVol.copyEncryptionConfig(&rv.rbdImage, false)
+		err = parentVol.copyEncryptionConfig(&rv.rbdImage, true)
```

Comment:
Destination pvc in case of restore/clone will have possibly different encryption config already initialized.